src/api/l_data_modelData.c

```diff
@@ -536,7 +537,7 @@ static int l_lovrModelDataGetAnimationChannelKeyframe(lua_State* L) {
   for (uint32_t i = 0; i < count; i++) {
```

Comment:
Hey so for the record what guarantees that channel->property <= 3?

Reply:
It's an enum